src/OpenTelemetry.Exporter.Console/ConsoleMetricExporter.cs

```diff
@@ -75,13 +75,13 @@ public override ExportResult Export(in Batch<MetricItem> batch)
 
                         case MetricType.LongGauge:
                             {
-                                // TODOs
+                                valueDisplay = (metric as IGaugeMetric).LastValue.Value.ToString();
```

Comment:
there are changes that are upcoming to this section. For now, this works (to demo the usage of Gauge), and the changes to this section are coming as separate PRs.